pkg/envoy/cds/response.go

```diff
@@ -66,8 +66,8 @@ func NewResponse(catalog catalog.MeshCataloger, proxy *envoy.Proxy, _ *xds_disco
 	}
 	clusterFactories[localCluster.Name] = localCluster
 
-	if cfg.IsEgressEnabled() {
-		// Add a pass-through cluster for egress
+	if cfg.IsEgressEnabled() || cfg.IsPermissiveTrafficPolicyMode() {
```

Comment:
@shashankram , I'm not sure this is correct. I understood permissive as allowing traffic both inmesh and outmesh. If this is not correct let me know.

Reply:
The passthrough cluster is a special cluster used specifically for egress, so the conditional should not include permissive mode.

Please revert this to as it was before.

Reply:
will do